BrWizardNavigator.vue

```diff
@@ -53,24 +53,62 @@ export default {
       required: true
     },
   },
+  data() {
+    return {
+      loadingBack: false,
+      loadingFinish: false,
+      loadingNext: false
+    };
+  },
+  computed: {
+    disableBack() {
+      return this.blockBack || this.loadingBack;
+    },
+    disableFinish() {
+      return this.blockFinish || this.loadingFinish;
+    },
+    disableNext() {
+      return this.blockNext || this.loadingNext;
+    }
+  },
   methods: {
     async onNext() {
-      if(!await this._emitCancelableEvent('next', {
-        currentStepIndex: this.currentStepIndex
-      })) {
-        return;
+      try {
+        this.loadingNext = true;
+        if(!await this._emitCancelableEvent('next', {
+          currentStepIndex: this.currentStepIndex
+        })) {
+          return;
+        }
+      } catch(e) {
+        console.error(e);
```

Comment:
I don't think we want an error logged here -- if someone wants to log it higher up when they reject they may.